projects/overlook.md

```diff
@@ -12,30 +11,47 @@ For this project, you will be building a hotel management tool for hotel custome
 
 ## Goals and Objectives
 
-- Use OOP to drive the design of the application and the code
-- Work with an API to send and receive data
-- Solidify the code review process
-- Create a robust test suite that thoroughly tests all functionality of a client-side application
+- Use object and array prototype methods to perform data manipulation
+- Create a clear and accessible user interface
+- Make network requests to retrieve data
+- Implement a robust testing suite using TDD
+- Write DRY, reusable code that follows SRP (Single Responsibility Principle)
+- Collaborate productively and professionally as a team. Ensure all team members are able to be heard and contribute throughout the project
```

Comment:
Since this is a solo final project I think we can remove line 19

Reply:
Great catch!